docs/modules/ROOT/pages/how-tos/multi-instance.adoc

```diff
@@ -0,0 +1,60 @@
+= Deploy multiple instances
+
+This guide provides an example how to deploy multiple instances of this component.
+
+== Requirements
+
+
+. Prepare catalog by configuring 2 instances
++
+[source,yaml]
+----
+applications:
+  - exoscale-metrics-collector as collector-ch-gva-2 # alias collector-ch-gva-2
+  - exoscale-metrics-collector as collector-rma-0 # alias collector-rma-0
```

Comment:
I have decided to cut the provider from the alias on purpose just to make sure it is no creating confusion for someone that is reading the doc...

Reply:
The naming scheme for a APPUiO Zone contains the provider: https://kb.vshn.ch/appuio-cloud/references/dns-naming-scheme.html#_zone
It's not about reading it, since the snippets here going to be copy-pasted anyway, so it should be copy-pastable. Without the provider, you're not really using the zone identifier correctly :shrug: